client/pom.xml

```diff
@@ -76,6 +76,14 @@
             <groupId>org.vectomatic</groupId>
             <artifactId>lib-gwt-svg</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
the version number is specified in the parent pom in the dependency management section.